src/main/java/org/gridsuite/modification/server/modifications/PropertiesUtils.java

```diff
@@ -1,52 +1,70 @@
 package org.gridsuite.modification.server.modifications;
 
-import com.powsybl.commons.report.ReportNode;
-import com.powsybl.commons.report.TypedValue;
+import com.powsybl.commons.report.*;
 import com.powsybl.iidm.network.Identifiable;
 import org.gridsuite.modification.server.dto.FreePropertyInfos;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 
 public final class PropertiesUtils {
+    public static final String PROPERTIES = "Properties";
 
     private PropertiesUtils() {
         // Should not be instantiated
     }
 
-    public static void applyProperties(Identifiable<?> identifiable, ReportNode subReportNode, @Nullable List<FreePropertyInfos> properties) {
+    public static void applyProperties(Identifiable<?> identifiable, ReportNode subReportNode, @Nullable List<FreePropertyInfos> properties, String propertiesLabelKey) {
+        List<ReportNode> reportNodes = new ArrayList<>();
+        ReportNode propertiesReporter = subReportNode.newReportNode().withMessageTemplate(propertiesLabelKey, PROPERTIES).add();
```

Comment:
```suggestion
        ReportNode propertiesReporter = subReportNode.newReportNode().withMessageTemplate(PROPERTIES, PROPERTIES).add();
```
So you do the same as it is done for setpoints or limits. And you need a new parameter in applyProperties()